### PR TITLE
Add jest and cypress tests for spans components

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*

*Description of changes:*
- add IT and UT for #44 changes
- CI should pass once #44 is merged

 By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by chloe-zh — APPROVED** [review]

**Review by mengweieric — APPROVED** [review]

File: test/constants.ts [hunk 1]
```diff
@@ -1,3 +1,141 @@
+export const TEST_SPAN_RESPONSE = {
```
Comment (davidcui-amzn) [inline test/constants.ts:1]: copyright notice missing

File: .cypress/integration/services.spec.js [hunk 2]
```diff
@@ -53,7 +61,16 @@ describe('Testing services table', () => {
 
 describe('Testing service view empty state', () => {
   beforeEach(() => {
-    cy.visit(`app/opendistro-trace-analytics#/services/${SERVICE_NAME}`);
+    // exception is thrown on loading EuiDataGrid in cypress only, ignore for now
```
Comment (davidcui-amzn) [inline .cypress/integration/services.spec.js:64]: Just want to verify based on this comment, this does not break any workflows right?

**Review by davidcui-amzn — APPROVED** [review]
LGTM! a few minor comments

File: .cypress/integration/services.spec.js (hunk 2 above)
Comment (joshuali925) [reply .cypress/integration/services.spec.js:64]: should not affect anything, seems to be a conflict with the data grid component and some nested flex items inside cypress. I couldn't see the error when testing manually

**Pull request merged**